### ISSUE TITLE
Bug: False Positives

### ISSUE THREAD

**Issue by lautarodragan** [judepereira/checktodo#5]
Check TODO is currently reporting the following lines as containing a TODO:

```
    src/daos/RegistryDao.ts#L19: const queryDocument = modelToDocument(query)
    src/daos/RegistryDao.ts#L25: const queryDocument = modelToDocument(query)
    src/daos/RegistryDao.ts#L31: const document = modelToDocument(registry) as RegistryDocument
    src/daos/RegistryDao.ts#L63: const modelToDocument = (model: Partial<Registry>): Partial<RegistryDocument> =
```
I think it's matching against _model**ToDo**cument_.

The PR can be seen at https://github.com/poetapp/frost-api/pull/1007/checks?check_run_id=412207324.

**Comment by jstarpl** [comment]
I've submitted a PR that should fix this: #6

**Comment by judepereira** [comment]
Closed with #6.
